CairoMakie/src/scatter.jl

```diff
@@ -4,7 +4,11 @@ function draw_atomic(scene::Scene, screen::Screen, plot::Scatter)
     Makie.add_computation!(attr, scene, Val(:meshscatter_f32c_scale))
     cairo_unclipped_indices!(attr)
     Makie.compute_colors!(attr)
-    project_to_markerspace!(attr)
+    Makie.register_positions_projected!(
```

Comment:
Why does a backend mutate the compute graph? Does that have side effects when switching back and forth between CairoMakie and GLMakie?

Reply:
It's thought to be just another "conversion".
We're prefixing backend specific conversions and try to re-use most conversions that aren't backend specific.